src/app/components/media/AudioMediaCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Player } from '@meedan/check-ui';
+import FormControl from '@material-ui/core/FormControl';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import AspectRatio from '../layout/AspectRatio';
+import { units } from '../../styles/js/shared';
+
+const StyledPlaybackRate = styled.div`
+  margin-top: ${units(2)};
+  position: absolute;
+  right: 0;
+`;
+
+const AudioMediaCard = (props) => {
+  const [playbackRate, setplaybackRate] = React.useState(1);
+
+  return (
+    <article className="video-media-card" style={{ position: 'relative' }}>
```

Comment:
wrong call name

Reply:
I kept same class name to apply same CSS for videos until decide a new design for audio

Reply:
Used `MediaPlayerCard` component for both video & audio